File: testdata/jennies/builders/map_of_builders/PHPBuilder/src/MapOfBuilders/PanelBuilder.php
<?php

namespace Grafana\Foundation\MapOfBuilders;

/**
 * @implements \Grafana\Foundation\Cog\Builder<\Grafana\Foundation\MapOfBuilders\Panel>
 */
class PanelBuilder implements \Grafana\Foundation\Cog\Builder
{
    protected \Grafana\Foundation\MapOfBuilders\Panel $internal;

    public function __construct()
    {
    	$this->internal = new \Grafana\Foundation\MapOfBuilders\Panel();
    }

    /**
     * Builds the object.
     * @return \Grafana\Foundation\MapOfBuilders\Panel
     */
    public function build()
    {
        return $this->internal;
    }

    public function title(string $title): static
    {
        $this->internal->title = $title;
    
        return $this;
    }

}

File: testdata/jennies/builders/map_of_builders/JavaBuilder/map_of_builders/PanelBuilder.java
package map_of_builders;


public class PanelBuilder implements cog.Builder<Panel> {
    protected final Panel internal;
    
    public PanelBuilder() {
        this.internal = new Panel();
    }
    public PanelBuilder title(String title) {
        this.internal.title = title;
        return this;
    }
    public Panel build() {
        return this.internal;
    }
}
